src/identity.js

```diff
@@ -1819,7 +1819,7 @@ export default function Identity(mpInstance) {
         deleted,
         user
     ) {
-        if (!previousUserAttributeValue) {
+        if (previousUserAttributeValue === undefined) {
```

Comment:
I think this is a better way to check to make sure that this variable is actually defined.

```suggestion
        if (typeof previousUserAttributeValue === 'undefined') {
```